addons/spreadsheet/static/src/helpers/model.js

```diff
@@ -140,17 +141,16 @@ function containsOdooFunction(content) {
     ) {
         return false;
     }
-    try {
-        const ast = parse(content);
-        return iterateAstNodes(ast).some(
-            (ast) =>
-                ast.type === "FUNCALL" &&
-                (ast.value.toUpperCase().startsWith("ODOO.") ||
-                    ast.value.toUpperCase().startsWith("_T"))
-        );
-    } catch {
+    const ast = parse(content);
```

Comment:
I think there's some confusion here and an over-enthusiastic grepping :smile:  This is the spreadsheet parser (which parses spreadsheet formulas and expressions), not the py parser :wink:

Reply:
my bad (but the test in community pass...)

Reply:
> but the test in community pass...

Thanks for pointing that out :) I'll add a test (https://github.com/odoo/odoo/pull/140418)